add query param to search registry records. (#1861)
plone/plone.restapi: docs/source/endpoints/registry.md

:language: http
```

> ## Filtering registry records

Reviewer: We'll need to update the version to whatever is next, which I think will be 9.10.0 based on [semver](https://semver.org/).

````suggestion
## Filter list of registry records

```{versionadded} plone.restapi 9.10.0
```
````

> Filtering registry records is done by passing a `q` parameter with the prefix of the record name. Filtering also supports {doc}`../usage/batching` like listing registry records.

Reviewer: ```suggestion
You can filter a list of registry records and batch the results.
To do so, append a query string to the listing endpoint with a `q` parameter and its value set to the prefix of the desired record name.
See {doc}`../usage/batching` for details of how to work with batched results.
```

:language: http
```

## Filter list of registry records

> 

Reviewer: ```suggestion
```
